maquetacion/src/scss/_navbar.scss

```diff
@@ -0,0 +1,31 @@
+.nav {
+  background-color: #FFF;
+  border-top: 6px solid #00ADEE;
+  box-shadow: 0 4px 10px 0 rgba(0,0,0,0.5);
+  position: fixed;
+  width: 100%;
+  top: 0;
+  z-index: 1;
+  
+  &__content {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 70px;
+  }
+
+  &__logo {
+    height: 70px;
+    width: 212.42px;
```

Comment:
Why decimals?

Reply:
Height remove and width changed to 215px